dotcom-rendering/scripts/webpack/webpack.config.client.js

```diff
@@ -19,21 +19,23 @@ const swcLoader = (targets) => [
 	},
 ];
 
+/** @typedef {import('../../src/lib/assets').Build} Build*/
+
```

Comment:
Equally, we could have gone with `Bundle` – I don’t have a strong preference, here, but I think these two concepts should be unified.